iot-hub/Samples/device/ImportExportDevicesSample/ImportExportDevices.csproj

```diff
@@ -0,0 +1,125 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{57DFE1E9-E86D-4C52-A82D-CEE289F5EBF2}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>ImportExportDevicesSample</RootNamespace>
+    <AssemblyName>ImportExportDevicesSample</AssemblyName>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Please switch to .NET Core 2.2 for all our samples.
We do not recommend new applications targeting old .NET Versions like 4.6.1.

Reply:
Also, please switch to the new .csproj format. It's simpler if you use the `dotnet new` command to generate the csproj.